js/console.go

```diff
@@ -62,7 +64,20 @@ func (c console) log(ctx *context.Context, level logrus.Level, msgobj goja.Value
 		}
 	}
 
-	msg := msgobj.String()
+	var msg string
+	switch msgobj.ExportType().Kind() { //nolint:exhaustive
```

Comment:
Are we planning to have more cases for the switch? If not, maybe it's better to extract this to a small method where use `if`s is possible?